docs/docs/References/CLI.md

```diff
@@ -76,7 +80,7 @@ $ dvp -vvv build
 Print nothing to the console.
 
 ```
-$ dvp -qqq upload -e engine.example.com -u admin
+$ dvp -qqq build
```

Comment:
what's the reason you changed this to build? just because it has default params?

Reply:
`dvp -qqq upload` and `dvp upload`
both had the exact same output, whereas `build` was different